webview/src/experiments/components/AddStage.tsx

```diff
@@ -0,0 +1,24 @@
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
+import React from 'react'
+import styles from './table/styles.module.scss'
+import { IconButton } from '../../shared/components/button/IconButton'
+import { Add } from '../../shared/components/icons'
+import { sendMessage } from '../../shared/vscode'
+
+export const AddStage: React.FC = () => (
+  <div className={styles.addConfigButton}>
+    <p>Want to easily and efficiently reproduce your experiments? </p>
```

Comment:
[C] I don't think we use this style anywhere else. Maybe make it a statement instead of a question. I.e `Easily and efficiently reproduce your experiments.` but up to you.